src/couch/test/eunit/couch_key_tree_tests.erl

```diff
@@ -534,7 +557,7 @@ should_not_use_excessive_memory_when_stemming() ->
     ?_test(begin
         % This is to preserve determinism
         Seed = {1647, 841737, 351137},
-        Tree = generate_rev_tree(1000, 0.006, Seed),
+        {_, Tree} = test_util:revtree_generate(1000, 0.06, 15, Seed),
```

Comment:
Nice!

Reply:
One intended effect of the change is to keep the functions in the `test_util` module, which is available in a running cluster so it allows running interactive benchmarks and such.